data/mods/ssb/statuses.js

```diff
@@ -1006,16 +978,28 @@ let BattleStatuses = {
 			this.add(`c|@pluviometer|${["Follow SmogonU on Facebook! https://www.facebook.com/SmogonU", "Follow SmogonU on Twitter! https://twitter.com/SmogonU"][this.random(2)]}`);
 		},
 	},
-	pre: {
+	pokemondeadchannel: {
 		noCopy: true,
 		onStart() {
-			this.add(`c|&pre|let's go, in and out, 20 minute adventure`);
+			this.add(`c|+PokemonDeadChannel|づ∧＿∧づﾀﾞﾗﾗﾗ`);
+			this.add(`c|+PokemonDeadChannel|づ｀･ω･)づ∴“`);
+			this.add(`c|+PokemonDeadChannel|づづづづづ∴≒`);
+			this.add(`c|+PokemonDeadChannel|づづづづづづ∵`);
+			this.add(`c|+PokemonDeadChannel|(_／⌒ヽ_)∴`);
+			this.add(`c|+PokemonDeadChannel|ORA ORA ORA ORA ORA !!!`);
 		},
 		onSwitchOut() {
-			this.add(`c|&pre|sometimes science is more art than science`);
+			this.add(`c|+PokemonDeadChannel|Bye Losers`);
+			this.add(`c|+PokemonDeadChannel|　＜⌒／ヽ-､_＿_`);
```

Comment:
would it possible to ignore this line specifically in the travis ci test? The irregular white space is necessary

Reply:
sure

Reply:
i see. enlighten me